vllm/lora/models.py

```diff
@@ -30,6 +31,9 @@
 
 _GLOBAL_LORA_ID = 0
 
+# NOTE This value comes from vllm/worker/model_runner.py
+_MAX_BATCH_SIZE = 256
```

Comment:
It looks like model_runner gets this from the scheduler config i.e. `self.scheduler_config.max_num_seqs`. Should we also get this user-defined value?

Reply:
`_MAX_BATCH_SIZE` is obtained from [_BATCH_SIZES_TO_CAPTURE](https://github.com/vllm-project/vllm/blob/main/vllm/worker/model_runner.py#L72), it appears to be a constant.  However, using `self.scheduler_config.max_num_seqs` is indeed more reasonable. It has been modified. Thank you.